docs/asciidoc/modules/ROOT/pages/overview/apoc.trigger/index.adoc

```diff
@@ -5,104 +5,124 @@ This file is generated by DocsTest, so don't change it!
 = apoc.trigger
 :description: This section contains reference documentation for the apoc.trigger procedures.
 
-== APOC new procedures 
+== Current procedures
+
 [.new procedures, opts=header, cols='5a,1a,1a']
 |===
 | Qualified Name | Type | Release
+
+|xref::overview/apoc.trigger/apoc.trigger.install.adoc[apoc.trigger.install icon:book[]]
+
+CALL apoc.trigger.install(databaseName, name, statement, selector, config) \| eventually adds a trigger for a given database which is invoked when a successful transaction occurs.
+|label:procedure[]
+|label:apoc-core[]
+
+
 |xref::overview/apoc.trigger/apoc.trigger.drop.adoc[apoc.trigger.drop icon:book[]]
 
 CALL apoc.trigger.drop(databaseName, name) \| eventually removes an existing trigger, returns the trigger's information
 |label:procedure[]
 |label:apoc-core[]
+
+
 |xref::overview/apoc.trigger/apoc.trigger.dropAll.adoc[apoc.trigger.dropAll icon:book[]]
 
 CALL apoc.trigger.dropAll(databaseName) \| eventually removes all previously added trigger, returns triggers' information
 |label:procedure[]
 |label:apoc-core[]
-|xref::overview/apoc.trigger/apoc.trigger.install.adoc[apoc.trigger.install icon:book[]]
 
-CALL apoc.trigger.install(databaseName, name, statement, selector, config) \| eventually adds a trigger for a given database which is invoked when a successful transaction occurs.
+
+|xref::overview/apoc.trigger/apoc.trigger.start.adoc[apoc.trigger.start icon:book[]]
+
+CALL apoc.trigger.start(databaseName, name) \| eventually unpauses the paused trigger
 |label:procedure[]
 |label:apoc-core[]
 
-|xref::overview/apoc.trigger/apoc.trigger.pause.adoc[apoc.trigger.pause icon:book[]]
 
-CALL apoc.trigger.pause(name) \| it pauses the trigger
+|xref::overview/apoc.trigger/apoc.trigger.stop.adoc[apoc.trigger.stop icon:book[]]
+
+CALL apoc.trigger.stop(databaseName, name) \| eventually pauses the trigger
 |label:procedure[]
 |label:apoc-core[]
 
+
 |xref::overview/apoc.trigger/apoc.trigger.show.adoc[apoc.trigger.show icon:book[]]
 
 CALL apoc.trigger.show(databaseName) \| it lists all eventually installed triggers for a database
 |label:procedure[]
 |label:apoc-core[]
+
+
+|xref::overview/apoc.trigger/apoc.trigger.list.adoc[apoc.trigger.list icon:book[]]
+
+CALL apoc.trigger.list() \| list all currently working triggers for all databases for the session database
```

Comment:
To be consistent, I would change `list all currently working triggers for all databases` to `it lists all currently working triggers for all databases`, as you rightly changed in `generated-documentation/documentation.adoc`.

In addition to changing the same wording in the `@Description` of Trigger.java (line 108), 
and in all other documentation files as well.

Reply:
Great catch, I've made an update in commit https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3378/commits/c2c22e1b9031bb6a93b56ffc88df7a9a70118211